build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.1.0"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.1.1"
   kotlin("plugin.spring") version "1.6.10"
```

Comment:
4.1.3 (and 4.1.4, I think) are available if you wanted to go later?

Reply:
according to the README the latest is 4.1.1
https://github.com/ministryofjustice/dps-gradle-spring-boot